### PR TITLE
support the new CLI in py2 *and* py3

### PR DESCRIPTION
**Pull request by ryanpetrello**

### CODE REVIEW

File: awxkit/requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 PyYAML
 requests
+six
```
Comment (ryanpetrello) [inline awxkit/requirements.txt:3]: This is a new dependency we'll have to require.

File: awxkit/awxkit/cli/resource.py [hunk 2]
```diff
@@ -121,12 +123,14 @@ def parse_resource(client, skip_deprecated=False):
             if k in ('dashboard',):
                 # the Dashboard API is deprecated and not supported
                 continue
-            aliases = []
-            if not skip_deprecated:
+
+            # argparse aliases are *only* supported in Python3 (not 2.7)
```
Comment (ryanpetrello) [inline awxkit/awxkit/cli/resource.py:127]: I want to follow-up and see if I can come up with another way to support this in py2

**Review by squidboylan — APPROVED** [review]
LGTM

File: awxkit/awxkit/utils/__init__.py [hunk 3]
```diff
@@ -320,6 +321,22 @@ def update_payload(payload, fields, kwargs):
     return payload
 
 
+def to_str(obj):
+    if six.PY3:
```
Comment (ryanpetrello) [inline awxkit/awxkit/utils/__init__.py:325]: This function is a little gross, and I suspect `six` has something to do this in a _cleaner_ way, but this is also code that I largely cherry-picked from the awxkit history from *before* the py2 -> py3 move (so I know it works and shouldn't break anything in py2).

File: awxkit/setup.py [hunk 4]
```diff
@@ -67,7 +67,7 @@ def run(self):
     },
     include_package_data=True,
     install_requires=requirements,
-    python_requires=">= 3.5",
+    python_requires=">=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*",
```
Comment (wenottingham) [inline awxkit/setup.py:70]: :nauseated_face: 

(not that I have a better suggestion)

Comment (ryanpetrello) [reply awxkit/setup.py:70]: Yea, from reading the related PEP, this is *apparently* the official blessed way to do this.

Gross.

**Pull request merged**